### ISSUE TITLE
RibbonOrbMenuItem firing when Orb is clicked and Ribbon is minimzed

### ISSUE THREAD

**Issue by xeredor** [RibbonWinForms/RibbonWinForms#44]
I'm using the Office_2013 Orb with the latest release. It seems that my first RibbonOrbMenuItem is firing a click event when I have the Ribbon minimzed and want to click on the Orb. 

If I hold down the mouse and slide just a few pixels away from the Orb, nothing get's fired. It's just in minimized state, when maximized/normal, everything works fine.

Edit: I tried a little bit, it seems that not my first OrbMenuItem is fired, it's the leftest Ribbon Button of the current open Ribbon Tab. So the Ribbon seems to be still open, as I can see the tooltips, and overlaying over my RibbonOrb...

If you need any additional info in replicating this issue, just send me a little message

**Comment by adriancs2** [comment]
Hi, I have made a quick test, and the problem you mentioned seem to not exist. I'm unsure of the problem at the moment. But anyway, I'm using the nuget version of the library, you might want to check out the nuget version. 

Below I have attached a sample project. You can try it out in your environment and see if the problem still exists.

[WindowsFormsApp2.zip](https://github.com/RibbonWinForms/RibbonWinForms/files/3479798/WindowsFormsApp2.zip)

**Comment by xeredor** [comment]
Hi, I've downloaded your sample and the effect was gone. So I searched for about half an hour in my Ribbon settings, just to find out I've set my RibbonButton "MinimumSize" to "60; 50". This way, the button behaves as it should (I think) and even sets the button size when the Ribbon is minimized. I will just delete the MinimumSize and everything is working as it should.

Thank you for your quick help.
